### DISCUSSION TITLE
Remote controlled robot with BLE communication

### DISCUSSION THREAD

**Discussion by afarago** [pybricks/support#1118] [Show and tell]
I really like the new BLE communication and gave it a test run for remote controlling a simple bot.

Gave it a test run on a family+children's day event recently (kids really loved it), and also my FLL team kids played around with it happily.

Functionality:
- car steering wheel mode
- airplane steering mode
- actuator motor control
- mutual connect / synchronization - RSSI display
- controlled robot heading feedback and display

Test run/demo:
https://photos.app.goo.gl/ZUTe8jEfDc8qqYLz6

Sample code:
https://gist.github.com/afarago/e19fdc2e0302fd2ef82cfbbd12377fde

Thanks for this great functionality!

**Comment by dlech** [comment]
Thanks for sharing!
